packages/api/src/middleware/validation/activeManagementPlanCheck.js

```diff
@@ -0,0 +1,29 @@
+
+const managementPlanModel = require('../../models/managementPlanModel');
+
+async function activeManagementPlanCheck(req, res, next) {
+  const currentDate = new Date();
+  const {body} = req;   // body should be for cropVarietyModel??
+  if (body.crop_variety_id) {
+    const doesCropVarietalHaveAManagementPlan = await managementPlanModel.query().where({crop_variety_id: body.crop_variety_id})// check if there is a management plan that has the crop varietal
+    if (doesCropVarietalHaveAManagementPlan !== null) { // crop varietal is in a management plan
+      if (body.seed_date === null) { // seed date is null
+        return next();
+      }
```

Comment:
This is not valid, but there is no need to check it here. Objection will check it for you